lib/ddtrace/pipeline.rb

```diff
@@ -9,6 +9,14 @@ module Pipeline
     @mutex = Mutex.new
     @processors = []
 
+    # {.before_flush} allows application to alter or filter out traces before they are flushed.
+    #
+    # @see file:docs/ProcessingPipeline.md Processing Pipeline
+    # @param [Array<Proc>] processors a list of callable objects that receive a list of {Datadog::Span}s and can modify
+    #   or filter our spans.
+    # @yield Optional that receives an array of spans and returns the desired remaining spans.
+    # @yieldparam [Array<Datadog::Span>] spans spans that can be modified or removed from list before flushing.
```

Comment:
Should we call the yield param `trace`, for consistency with what we document above?

Reply:
Rewritten this part, should be consistent now.